libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/DBSchemaAccessor.java

```diff
@@ -198,6 +198,12 @@ default List<String> showTables(String schemaName) {
      */
     List<DBTableColumn> listBasicViewColumns(String schemaName, String viewName);
 
+    /**
+     * Get all table and view columns(hold only basic info: schema, table and column name) in the
+     * specified schema
+     */
+    Map<String, List<DBTableColumn>> listBasicColumns(String schemaName);
```

Comment:
name suggestion: listBasicTableColumnsInfo

Reply:
Okay, I renamed it to `listBasicColumnsInfo`, this method will return both view and table columns.